wiremock/src/goggles.ts

```diff
@@ -42,7 +42,7 @@ launch({
     `--proxy-server=${httpProxy}`,
     `--proxy-bypass-list=${httpProxyBypassList}`,
     '--args',
-    '--user-data-dir="/tmp/chrome_dev_test"',
+    '--user-data-dir=/tmp/chrome_dev_test',
```

Comment:
Another option would be to put the user data dir inside `./.cache` and thus contain it within the project.